cartography/models/semgrep/findings.py

```diff
@@ -68,6 +68,40 @@ class SemgrepSCAFindingToGithubRepoRel(CartographyRelSchema):
     properties: SemgrepSCAFindingToGithubRepoRelProperties = SemgrepSCAFindingToGithubRepoRelProperties()
 
 
+@dataclass(frozen=True)
+class SemgrepSCAFindngToDependencyRelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SemgrepSCAFinding)-[:AFFECTS]->(:Dependency)
+class SemgrepSCAFindingToDependencyRel(CartographyRelSchema):
+    target_node_label: str = 'Dependency'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('matchedDependency')},
+    )
+    direction: LinkDirection = LinkDirection.OUTWARD
+    rel_label: str = "AFFECTS"
+    properties: SemgrepSCAFindngToDependencyRelProperties = SemgrepSCAFindngToDependencyRelProperties()
+
+
+@dataclass(frozen=True)
+class SemgrepSCAFindingToCVERelProperties(CartographyRelProperties):
+    lastupdated: PropertyRef = PropertyRef('lastupdated', set_in_kwargs=True)
+
+
+@dataclass(frozen=True)
+# (:SemgrepSCAFinding)<-[:LINKED_TO]-(:CVE)
+class SemgrepSCAFindingToCVERel(CartographyRelSchema):
+    target_node_label: str = 'CVE'
+    target_node_matcher: TargetNodeMatcher = make_target_node_matcher(
+        {'id': PropertyRef('cveId')},
+    )
+    direction: LinkDirection = LinkDirection.INWARD
+    rel_label: str = "LINKED_TO"
```

Comment:
rename to something else like CAUSES

Reply:
Sorry to push back, but `:CVE` nodes do not cause `:SemgrepSCAFinding`.
A `:SemgrepSCAFinding` is a result from a scan that might or might not have attached CVE information.
Hence, the CVE is not causing the finding, but rather the CVE is connected to the finding.
Other findings from different tools might be as well connected to the same CVE.